Recovery RFCEditor #3: Why reset min_rtt (#4891)
quicwg/base-drafts: rfc9002.md

future RTT samples that are smaller than the new RTT to be included in
smoothed_rtt.

> Endpoints SHOULD set the min_rtt to the newest RTT sample after persistent

Reviewer: ```suggestion
Endpoints SHOULD set the min_rtt to the newest RTT sample after persistent
```

Author: These two lines look identical, but I expect they're not and I copied over a non-breaking space or something?

> congestion is established because it is possible that an increase in path delay
> resulted in persistent congestion being incorrectly declared. This also allows

Reviewer 2: ```suggestion
congestion is established. This helps when an increase in path delay
results in persistent congestion being incorrectly declared. This also allows
```

a connection to reset its estimate of min_rtt and smoothed_rtt after a

> disruptive network event (Section 5.3).

Reviewer: ```suggestion
disruptive network event; see {{smoothed-rtt}}.
```

smoothed_rtt.

Endpoints SHOULD set the min_rtt to the newest RTT sample after persistent

> congestion is established. This helps when an increase in path delay

Reviewer: Helps what?

The key is that it avoids repeatedly entering persistent congestion if the path RTT has genuinely increased.

> congestion is established. This helps when an increase in path delay
> results in persistent congestion being incorrectly declared. This also allows

Author: ```suggestion
congestion is established. This avoids repeatedly declaring persistent congestion
incorrectly when the RTT increases. This also allows
```

> congestion is established. This helps when an increase in path delay
> results in persistent congestion being incorrectly declared. This also allows
> a connection to reset its estimate of min_rtt and smoothed_rtt after a
> disruptive network event; see {{smoothed-rtt}}.

Reviewer 3: ```suggestion
congestion is established. This helps a connection deal when an increase
in path delay results in persistent congestion being incorrectly declared.
This also resets a connection's estimates of min_rtt and smoothed_rtt after
a disruptive network event; see {{smoothed-rtt}}.
```
